# USE ONLY WHEN THE OPTION COMPILE_WITH_OLD_ARCHITECTURE IS SET TO FALSE.
# https://github.com/autowarefoundation/autoware.universe/blob/main/planning/behavior_path_planner/CMakeLists.txt
# NOTE: The smaller the priority number is, the higher the module priority is.
/**:
  ros__parameters:
    ext_request_lane_change_left:
      enable_module: false
      enable_simultaneous_execution: false
      priority: 6
      max_module_size: 1

    ext_request_lane_change_right:
      enable_module: false
      enable_simultaneous_execution: false
      priority: 6
      max_module_size: 1

    lane_change_left:
      enable_module: true
      enable_simultaneous_execution: true
      priority: 5
      max_module_size: 1

    lane_change_right:
      enable_module: true
      enable_simultaneous_execution: false
      priority: 4
      max_module_size: 1

    pull_out:
      enable_module: true
      enable_simultaneous_execution: false
      priority: 0
      max_module_size: 1

    side_shift:
      enable_module: true
      enable_simultaneous_execution: false
      priority: 2
      max_module_size: 1

    pull_over:
      enable_module: true
      enable_simultaneous_execution: false
      priority: 1
      max_module_size: 1

    avoidance:
      enable_module: true
      enable_simultaneous_execution: false
      priority: 3
      max_module_size: 1
